lesson_15/tests/test_polyon.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from lesson_15.lesson_15_mihaela_vaida import polygon
+
+
+class TestPolygon(unittest.TestCase):
+    def test_polygon_object_len_of_sides(self):
+        sides = 4, 3, 4, 5
+        test_polygon = polygon.Polygons(sides)
+        self.assertGreaterEqual(len(sides),3, "Not a valid polygon!")
+
+    def test_polygon_object_sides_type(self):
+
+       test_polygon = polygon.Polygons(3, 4, 5)
```

Comment:
same as above you need to test the result after you run your code on the input. In this case, you need to make the test of test_polygon.sides[0] ( this will be 3),  test_polygon.sides[1] ( this will be 4),  test_polygon.sides[2] ( this will be 5)

Reply:
modified !